frontend/components/new/new-droplet-form.tsx

```
@@ -97,10 +107,74 @@ export function CreateDropletForm({
+                    <p className="text-sm font-medium text-red-800 dark:text-red-200">
+                      A published droplet with this name already exists.
+                    </p>
+                    <Link
+                      href={`/d/${slug}`}
+                      className="mt-2 inline-flex items-center gap-1 text-sm text-red-700 hover:text-red-900 dark:text-red-300 dark:hover:text-red-100"
+                    >
+                      View existing droplet "{dropletName.trim()}"
+                      <ExternalLink className="h-3 w-3" />
+                    </Link>
+                  </div>
+                ) as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)